src/strategy/strategy.ts

```diff
@@ -17,4 +17,6 @@ export class Strategy {
 export interface StrategyTransportInterface {
     name: string;
     parameters: any;
+    operator?: string;
+    group?: StrategyTransportInterface[];
```

Comment:
Would it make sense to define a group-interface that extends the `StrategyTransportInterface` instead?

```typescript
export interface StrategyGroupTransportInterface extends StrategyTransportInterface {
    operator: string;
    group: StrategyTransportInterface[];
}
```

Reply:
After revisiting the spec I realize that the operator should be on the `StrategyTransportInterface` and needs to be optional.

Reply:
👍